lib/mongoid/tasks/database.rb

```diff
@@ -11,10 +11,13 @@ module Database
       # @param [ Array<Mongoid::Document> ] models. Array of document classes for
       #   which collections should be created. Defaulted to all document classes
       #   in the application.
-      def create_collections(models = ::Mongoid.models)
+      # @param [ true | false ] force If true, the method will drop existing
+      #   collections before creating new ones. If false, the method will create
+      #   only new collection (that do not exist in the database).
+      def create_collections(models = ::Mongoid.models, force: false)
```

Comment:
I would rename this variable to `:drop_existing_collections`. The name force is not 100% that it will drop your collections, and given the potential severity of a mistake here, we should be 100% clear.

Reply:
A similar method in ActiveRecord they use[force](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-create_table), hence `force` here. I would leave it as it is now.

However, I agree that `drop_existing_collections` might be more explicit. @p-mongo @Neilshweky what do you think?

Reply:
The behavior is clearly described and matches Rails behavior. I don't see the need to change it.

Reply:
If Rails is using `force` then OK.

![image](https://user-images.githubusercontent.com/27655/188198260-e2a2ff49-e922-4aee-b722-e53c9a54397f.png)

Reply:
The point of the cartoon is exactly opposite of what you are requesting @johnnyshields .

Reply:
I just think about this cartoon every time I hear dropping tables.